docs/components/modeler/web-modeler/process-application-pipeline.md

```diff
@@ -0,0 +1,92 @@
+---
+id: process-application-pipeline
+title: Process application development pipeline
```

Comment:
In this context, I define the process development lifecycle (PDLC) as all activities that go from discovery to improvement, were some activities like monitoring or analysis are performed in a different app like Operate or Optimize
I define the `deployment pipeline` as the activities to get a release candidate through the different stages/environments and quality gates out into production